wireapp/wire-android-reloaded
# feat: enable adding multiple accounts

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

enable adding multiple accounts and navigate to `WelcomeScreen` with the user self server config

Needs releases with:


### Testing

#### Test Coverage (Optional)


#### How to Test

login -> open profile screen -> click on New Team or Account -> the WelcomeScreen should open with the user self server config

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
